documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1906,6 +1909,27 @@ Support for further data types is planned for subsequent releases.
 Please file a {jira-url}/browse/DBZ[JIRA issue] for any specific types that might be missing.
 endif::community[]
 
+[id="oracle-unsupported-types"]
+=== Unsupported data types
+
+The {prodname} Oracle connector does not support the full range of data types that are used in Oracle.
+The connector cannot process columns that include the following types of data:
+
+* BFILE (Binary file)
+* BOOLEAN. For more information, see xref:oracle-boolean-types[Boolean types].
+* Direct Joins in the FROM clause of UPDATE and DELETE statements
+* Data Use Case Domains
+* JavaScript-based stored procedures
+* LONG
+* LONG RAW
+* ROWID (not supported on XStream)
+* Table value constructor (TVCs)
+* UROWID (Universal ROWID)
+* VECTOR
+
+If the connector attempts to capture an event that includes a column with one of these unsupported data types, it logs a warning, and skips the column with the unsupported type.
+The connector continues to process other columns and tables, but data from the unsupported column is lost.
```

Comment:
```suggestion
If the connector attempts to capture an event that includes a column with one of these unsupported data types, it logs a warning, and skips the event.
```
Unforunately most of these generate an `UNSUPPORTED` event type in LogMiner and when this happens, it returns absolutely nothing about the event, not even the other columns :disappointed: